playbooks/archive_artifacts.yml

```diff
@@ -0,0 +1,77 @@
+- hosts: "{{ hosts }}"
+  tasks:
+    - name: Create temporary directory to store artifacts
+      file:
+        path: "{{ artifacts_path }}"
+        state: directory
+      delegate_to: "localhost"
+      run_once: yes
+
+    - name: Grab host data
+      command: >
```

Comment:
Fine for me, but could you add:
``
tags:
  - skip_ansible_lint
``

for the future, and a todo item to replace that with synchronize module then? 
I am afraid that it should pass the linting right now, so maybe we are already too lenient in our testing.